checkers/discover/fixtures/checkers/no_assert.go

```diff
@@ -0,0 +1,26 @@
+//registry:exclude
```

Comment:
That's not the way I would use for Go.

This would be more likely to sound like something expect for the language.

Please look at the spec

> A directive comment is a line matching the regular expression //(line |extern |export |[a-z0-9]+:[a-z0-9]). Tools that define their own directives should use the form //toolname:directive.
> Source: https://tip.golang.org/doc/comment#syntax

Think about revive, gosec, or nolint directives.

So I would more likely to expect something like these

```suggestion
//globstar:exclude
```

But one of these would be better I think

```suggestion
//globstar:exclude-file
```

```suggestion
//globstar:exclude-registry
```

Reply:
Hey @ccoVeille, thanks for the suggestion. It does seem from the Go docs that we should use the `//toolname:directive` format. I think we should use the third option, `//globstar:exclude-registry`.

Reply:
LGTM 👍